_overviews/scala3-book/types-opaque-types.md

```diff
@@ -98,8 +97,12 @@ Programming against the abstract module `Logarithms` can be very tedious and oft
 def someComputation(L: Logarithms)(init: L.Logarithm): L.Logarithm = ...
 ```
 
+#### Boxing Overhead
+Type abstractions, such as `type Logarithm` erase to their bound (which is `Any` in our case).
```

Comment:
Could we add a link around “erase” to some page somewhere that describes what type erasure is?